.env-template

```diff
@@ -1,2 +1,12 @@
 MYRIAD_WEBSITE_URL=
```

Comment:
change this to `NEXT_PUBLIC_MYRIAD_WEBSITE_URL` and put this below `NEXT_PUBLIC_MYRIAD_RPC_URL`
and add this to `next.config.ts` we'll need this